app/views/CapitalGainsView.scala.html

```diff
@@ -18,139 +18,164 @@
 @import uk.gov.hmrc.play.partials.FormPartialRetriever
 @import config.ApplicationConfig
 @import utils.ViewUtils._
-@import views.html.includes.SaMainView
 @import views.html.includes.CurrencyAmountView
-@import views.html.includes.TaxsBreadcrumbView
-@import views.html.helpers.BackLinkView
-
-@this(saMainView: SaMainView, taxsBreadcrumbView: TaxsBreadcrumbView, backLinkView: BackLinkView, currencyAmountView: CurrencyAmountView)
-
-@(viewModel: view_models.CapitalGains, actingAsAttorney: Option[ActingAsAttorneyFor] = None)(implicit request: AuthenticatedRequest[_], messages: Messages, formPartialRetriever: FormPartialRetriever, appConfig: ApplicationConfig)
-
-@headerContent = {
-  @taxsBreadcrumbView(List("breadcrumbs.taxs.select_tax_year", "breadcrumbs.taxs.index", "breadcrumbs.taxs.summary"), "ats.capital_gains_tax.subtitle", viewModel.taxYear)
-
-  @if(!actingAsAttorney.isDefined){
-    <p id="user-info"><strong>@viewModel.forename @viewModel.surname</strong> <br>@Html(Messages("generic.utr")) @viewModel.utr</p>
-  }
+@import views.html.main
+@import views.html.helpers.HeaderSection
+
+@this(
+    main: main,
+    headerSection: HeaderSection,
+    currencyAmountView: CurrencyAmountView
+)
+
+@(  viewModel: view_models.CapitalGains,
+    actingAsAttorney: Option[ActingAsAttorneyFor] = None
+    )(
+    implicit request: AuthenticatedRequest[_],
+    messages: Messages,
+    formPartialRetriever: FormPartialRetriever,
+    templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer,
+    appConfig: ApplicationConfig
+)
+
+@main(
+        title = Messages("ats.capital_gains_tax.html.title") + Messages("generic.to_from", viewModel.taxYearFrom, viewModel.taxYearTo),
+        actingAttorney = actingAsAttorney,
+        articleClasses = Some("full-width")
+    ) {
+
+
+    @headerSection(
+        pageHeading = Messages("ats.capital_gains_tax.subtitle"),
+        name = viewModel.forename + ' ' + viewModel.surname,
+        utr = viewModel.utr,
+        taxYear = viewModel.taxYear,
+        actingAsAttorney = actingAsAttorney,
+        breadcrumbLinks = List(
+            "breadcrumbs.taxs.select_tax_year",
+            "breadcrumbs.taxs.index",
+            "breadcrumbs.taxs.summary"
+        ),
+        currentBreadcrumbLink = "ats.capital_gains_tax.subtitle"
+    )
+
+    <div class="main-content">
+
+
+        <h1 class="heading-xlarge">
+            <span class="gray heading-medium block-heading">
+              @Html(Messages("generic.tax_year_heading", viewModel.taxYearFrom, viewModel.taxYearTo))
+            </span>
+          @Html(Messages("ats.capital_gains_tax.subtitle"))
+        </h1>
+        <p class="lede">@Html(Messages("ats.capital_gains_tax.lede"))</p>
+        @if(!viewModel.totalCapitalGainsTaxAmount.isZero) {
+          <p id="total-cg-description">@Html(Messages("ats.capital_gains_tax.description",
+            """<span id="total-cg-tax-rate">"""+viewModel.totalCgTaxRate.percent+"""</span>"""))</p>
```

Comment:
can you use plays StringInterpolation -> https://www.playframework.com/documentation/2.6.x/ScalaTemplates